source/administration/config-settings.rst

```diff
@@ -4159,25 +4163,50 @@ Specify the color of the SAML login button text for white labeling purposes. Use
 +-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
-Sidebar Organization (Experimental)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Enable Legacy Sidebar
+^^^^^^^^^^^^^^^^^^^^^
+
+*Not available in Mattermost Cloud. Available in Mattermost Server v5.32 and later.*
+
+This setting re-enables the legacy sidebar functionality for all users on this server. We strongly recommend that you leave this configuration setting disabled. The new sidebar offers most of the same functionality available in the legacy sidebar and includes a number of new features that make channel management easier.
+
+**Disabled**: The new sidebar functionality is enabled including custom collapsible channel categories, multi-channel select, drag, drop, and move channel organization, channel and category sorting, and more. This is the default setting.
+
+**Enabled**: When enabled, the legacy sidebar features are enabled for all users on this server. However, Mattermost users can’t disable legacy sidebar functionality in **Account Settings**.
+
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"EnableLegacySidebar": disabled`` with options ``enabled``, and ``disabled``.                                                            |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Sidebar Organization
+^^^^^^^^^^^^^^^^^^^^
+
+This setting applies to the legacy sidebar only. You must enable the `Enable Legacy Sidebar <https://docs.mattermost.com/administration/config-settings.html#enable-legacy-sidebar>`__ configuration setting to see and enable this functionality in the System Console.
+
+.. note::
+
+  We strongly recommend that you leave the **Enable Legacy Sidebar** configuration setting disabled. The new sidebar offers most of the same functionality available in the legacy sidebar, and includes a number of new features that make channel management easier.
 
-**True**: Enables channel sidebar organization options in **Account Settings > Sidebar > Channel grouping and sorting** including options for grouping unread channels, sorting channels by most recent post and combining all channel types into a single list.
+**True**: Enables channel sidebar organization options in **Account Settings > Sidebar > Channel grouping and sorting**. Includes options for grouping unread channels, sorting channels by most recent post, and combining all channel types into a single list.
 
-**False**: Hides the channel sidebar organization options in **Account Settings > Sidebar > Channel grouping and sorting**.
```

Comment:
please leave this as it was, this is still accurate if the legacy sidebar is enabled